src/vs/workbench/contrib/terminal/browser/xterm/quickFixAddon.ts

```diff
@@ -99,7 +129,27 @@ export class TerminalQuickFixAddon extends Disposable implements ITerminalAddon,
 		if (!terminal || !commandDetection) {
 			return;
 		}
-		this._register(commandDetection.onCommandFinished(command => this._resolveQuickFixes(command)));
+		this._register(commandDetection.onCommandFinished(command => {
+			if (this._fixesAvailable) {
+				if (this._fixesShown) {
+					if (this._expectedCommands?.includes(command.command)) {
+						this._logService.debug(QuickFixCommandResultTelemetryEvent.QuickFixExecutedManually);
+						this._telemetryService?.publicLog2<{ classification: 'SystemMetaData'; purpose: 'FeatureInsight' }>(QuickFixCommandResultTelemetryEvent.QuickFixExecutedManually);
```

Comment:
This didn't seem to work for me, running `--set-upstream` manually showed it as ignored, not executed manually:

![image](https://user-images.githubusercontent.com/2193314/195664740-1421115d-97fc-408f-aee9-504ec7cd53d0.png)

Reply:
if you didn't click on the lightbulb, that's what we consider `ignored`. `executedManually` only differs in that the lightbulb was consulted

Reply:
maybe instead of these 4 states it would be better if we had one for if the quick fix was interacted with / shown and another for if the next command run matched one of our suggestions

Reply:
Oh that confused me a little, how about instead it's a single command but we add whether the quick fix was opened as a common property to all the events? That way there's one executedManually but we can filter whether or not we care about whether it was opened.